op-batcher/batcher/driver.go

```diff
@@ -681,14 +688,17 @@ func (l *BatchSubmitter) clearState(ctx context.Context) {
 			}
 		case <-ctx.Done():
 			l.Log.Warn("Clearing state cancelled")
+			l.channelMgrMutex.Lock()
 			l.channelMgr.Clear(eth.BlockID{})
+			l.channelMgrMutex.Unlock()
```

Comment:
```suggestion
			l.channelMgrMutex.Lock()
			defer l.channelMgrMutex.Unlock()
			l.channelMgr.Clear(eth.BlockID{})
```
you can do this since you return right after

Reply:
[2bcc0ae](https://github.com/ethereum-optimism/optimism/pull/13330/commits/2bcc0ae86aadc9029c099973bb55c149efe1f189)